Bottom: 8975e1031edc1c6c4ef1a9542d81d31e934bbef3
Top:    927d8424df1eff935733aa2931b075903bf7d89e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 21:05:50 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 16247a8..385351e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
